docs/csharp/language-reference/builtin-types/ref-struct.md

```diff
@@ -30,15 +28,32 @@ To declare a `ref struct` as `readonly`, combine the `readonly` and `ref` modifi
 
 In .NET, examples of a `ref struct` are <xref:System.Span%601?displayProperty=nameWithType> and <xref:System.ReadOnlySpan%601?displayProperty=nameWithType>.
 
+## `ref` fields
+
+Beginning with C# 11, you can declare a `ref` field in a `ref struct`, as the following example shows:
+
+:::code language="csharp" source="snippets/shared/StructType.cs" id="SnippetRefField":::
+
+A `ref` field may have the `null` value. Use the <xref:System.Runtime.CompilerServices.Unsafe.IsNullRef%601?displayProperty=nameWithType> method to determine if a `ref` field is `null`.
+
+You can apply the `readonly` modifier to a `ref` field in the following ways:
```

Comment:
I'd add a sentence here, or at the end of this section, that the `readonly` keyword impacts the token to its right. In other words, `readonly ref` means you can't `ref` reassign. `ref readonly int n` means you can't reassign the value of `n`.

Reply:
@pkulikov I see your point. I'll approve this, and :shipit: 

I'll keep an eye on any feedback to see if we want refinements later.